rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/util/LocalJobFriendlyPackerCommandFactory.groovy

```diff
@@ -32,6 +31,7 @@ class LocalJobFriendlyPackerCommandFactory implements PackerCommandFactory {
     }
 
     packerCommand << absoluteTemplateFilePath
+    packerCommand.removeAll([null])
```

Comment:
I did a test of the join function in rush. This test is successful, I imagine null values to rush can break stuff:
```
  void 'test the string'() {
    when:
      List<String> listOfStrings = [ "test ", "", "empty", "and", null, "null", "strings" ]

    then:
      String output = listOfStrings.join(" ")

    expect:
      output == "test   empty and null null strings"

  }
```